### DISCUSSION TITLE
Training with virials

### DISCUSSION THREAD

**Discussion by Sokseiha** [mir-group/nequip#312] [Q&A]
Hi,

I've been training NequIP with energies and forces from DFT and so far, everything works very nicely. However, recently, I started to add virial in the training and things become, what should I say, strange. 
As you can see from the learning cures, the  the validation error for the virial plateaus while the training error keeps decreasing. I suspect that it has to do with the coefficients in the loss function so I've tried a few different values but that doesn't help. I'm wondering if any of you have seen this behavior before and if so, I'd really appreciate if you can suggest some ideas I can try to fix this problem.

Cheer.

![learing_curve](https://user-images.githubusercontent.com/20935064/221825307-fe95de7e-ff41-4db2-b063-077f72c241c4.png)

**Comment by Linux-cpp-lisp** [comment]
Hi @Sokseiha ,

This just looks like the virial overfits sooner than the energy and forces.  Which model gets picked will be determined by the best model on `validation_loss`, so yes, the balance will affect that.  You can try weighting virial lower (virial is an extensive property, so `PerAtom` loss should also be used).

But I have seen this before, it doesn't look insane.
